assignments/solutions/rustlatin/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "rustlatin"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
Should probably have a link here back to the exercise name, at least. Perhaps in the `description` field.